### DISCUSSION TITLE
[Live Competition] Test Run is not working?

### DISCUSSION THREAD

**Discussion by a4rcvv** [yasserfarouk/scml#118] [Q&A] [answered]
I have noticed that the test run result of my agent is not updated after I updated source codes for several times. It seems that Test Run is not working or the result is not updated.

<img width="481" alt="スクリーンショット 2023-04-24 14 58 46" src="https://user-images.githubusercontent.com/35623957/233911709-b456f1ca-77ba-4865-bbd1-a4286a3f1636.png">

**Comment by yasserfarouk** [comment, marked as answer]
The test runner was down because one agent overflowed the server memory. We restored it and tests should be finishing in a FIFO manner now. Please check again within 2-3 hours.

**Comment by a4rcvv** [comment]
I confirmed that the test run has completed. Thank you.
